collect_app/src/main/java/org/odk/collect/android/formhierarchy/FormHierarchyActivity.java

```diff
@@ -924,4 +927,11 @@ public void deleteGroup() {
             goUpLevel();
         }
     }
+
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+
+        CollectStrictMode.enable();
```

Comment:
Is this a temporary solution? If so should we file an issue to fix it in the future?

Reply:
I've added an issue: https://github.com/getodk/collect/issues/6470